src/bridging/types.ts

```diff
@@ -69,6 +69,28 @@ export interface BridgeQuoteResult {
    * The timestamp of the quote.
    */
   quoteTimestamp: number
+
+  fees: {
+    /**
+     * The amount that should go to the relayer as a fee to cover relayer capital costs.
+     */
+    bridgeFee: bigint
+    /**
+     * The amount that should go to the relayer as a fee to cover relayer gas costs.
+     */
+    destinationGasFee: bigint
+  }
+
+  limits: {
+    /**
+     * The minimum amount that should be deposited in the source chain.
+     */
+    minDeposit: bigint
+    /**
+     * The maximum amount that can be deposited in the source chain.
+     */
+    maxDeposit: bigint
```

Comment:
What units are they supposed to be in?
FIAT, sell token, buy token?
In case sell/buy tokens, in units or atoms?

Reply:
In token atoms.
This is info from bridge provider, in our case it's Across.
For now we only use canonical token, so it's the same for sell / buy.

Reply:
Added more comments